### ISSUE TITLE
feature-request: replace info-block directive with mdx component

### ISSUE THREAD

**Issue by EiffelFly** [instill-ai/instill.tech#496]
## Context

Currently, we are using info-block directive in markdown to implement components like this.

![截圖 2023-03-01 下午2 15 12](https://user-images.githubusercontent.com/57251712/222059770-b20a5b78-93da-494b-ac19-c654af5ba5ee.png)

But since we already adopt mdx in our codebase, we can safely replace this directive with mdx component.

### Directive we are using

https://github.com/instill-ai/instill.tech/blob/978978250b1a128ed8c5ae95359cc02dd89f4302/src/lib/markdown/remark-info-block.mjs#L5-L55

### The info block style

https://github.com/instill-ai/instill.tech/blob/978978250b1a128ed8c5ae95359cc02dd89f4302/src/styles/docs.css#L220-L456

### The mdx component reference

https://github.com/instill-ai/instill.tech/blob/978978250b1a128ed8c5ae95359cc02dd89f4302/src/components/ui/MdxQuoteBlock.tsx#L1-L18

**Comment by iamnamananand996** [comment]
I have started doing changes for this issue.
cc - @EiffelFly

**Comment by EiffelFly** [comment]
@iamnamananand996 Thanks you! Please proceed. If you have any question you could find me on Discord

**Comment by iamnamananand996** [comment]
@EiffelFly , I have made the bellow syntax for InfoBlock will it be okay. Please confirm this. so I can make this changes for all
`:::info{type=""}`

```javascript

<InfoBlock
  type="info"
  title="info"
  >
In above example, the generated images is a list of Base64 encoded images. To obtain the images, we need to decode Base64 as below snippet code.

```python
import base64
import numpy as np

# Decode the first image result

base64_image = out['text_to_image']['images'][0]
image = base64.b64decode(base64_image)

# Save the decoded image

filename = 'text_to_image.jpg'
with open(filename, 'wb') as f:
f.write(image)
``

</InfoBlock>


```

**Comment by EiffelFly** [comment]
@iamnamananand996 I am curious about that how this syntax parse the markdown string of the string. Are you using something like react-markdown?

**Comment by EiffelFly** [comment]
this issue should be closed with #513
